json-schemas/mngsWorkflowResults.json

```diff
@@ -104,7 +104,7 @@
                         "type": "object",
                         "properties": {
                             "tax_id": {
-                                "type": "integer"
+                                "type": "string"
```

Comment:
Align to schema returned by resolver.  This was preventing the tests from running, because the schema could not be generated